dspace-cris/jspui-api/src/main/java/org/dspace/app/webui/cris/controller/json/MyRPJSONController.java

```diff
@@ -64,6 +65,10 @@ public ModelAndView create(HttpServletRequest request,
         if (rp == null)
         {
             rp = new ResearcherPage();
+            String newRpStatus = ConfigurationManager.getProperty("cris","rp.profile.new.status");
```

Comment:
this can be simplified. You don't need the if case at all, please use
org.dspace.core.ConfigurationManager.getBooleanProperty(String, String, boolean)

org.dspace.core.ConfigurationManager.getBooleanProperty("cris", "rp.profile.new.status", false)

Reply:
Thank you for your comment and change request! Sounds good. I have implemented your suggestion into the PR.